app/controllers/users_controller.rb

```diff
@@ -45,4 +57,8 @@ def user_params
   def edit_user_params
     params.permit(:name, :address, :city, :state, :zip, :email)
   end
+
+  def password_params
```

Comment:
Needed another params to allow just the password to be updated (as password is updated on a separate page from other user info.